### ISSUE TITLE
suggestion: detect when using DiskMaster software with DiskBuddy

### ISSUE THREAD

**Issue by ajwood1965** [wagiminator/C64-Collection#14]
I spent a day trying to figure out why my DiskBuddy64 didn't work and finally realized i was using the DiskMaster64 folder, not the DiskBuddy64.   I suggest you detect this case and report an appropriate error.

thanks

**Comment by wagiminator** [comment]
Yes, I actually wanted to create a uniform software for all devices that would then automatically recognize which device was connected. But at the moment I don't have the time for it. Maybe I'll at least do the error message story at some point.
